wgpu/Cargo.toml

```diff
@@ -22,3 +23,8 @@ glam = "0.8"
 font-kit = "0.4"
 log = "0.4"
 resvg = { version = "0.8", features = ["raqote-backend"], optional = true }
+guillotiere = "0.4"
+debug_stub_derive = "0.3"
+
+[build-dependencies]
+shaderc = "0.6"
```

Comment:
I'd like to keep dependencies at bay. `shaderc` is quite a heavy dependency just to build some shaders. Forcing every contributor/user to compile them from source feels wrong.

I feel our current approach is better: recompile the shaders manually and version control the SPIR-V files. Of course, if you are working on shaders heavily, you are free to create a local `Makefile` or whatever adapts well to your workflow in order to increase productivity.

We should also implement `Debug` manually instead of relying in a dependency with a custom derive macro.